docker/protect.json

```diff
@@ -1,36 +1,147 @@
 {
```

Comment:
That was not evident in this. Add a multi-line comment at teh top stating this, and add it to the README